tests/storage/hotplug.go

```diff
@@ -469,9 +467,14 @@ var _ = SIGDescribe("Hotplug", func() {
 		if volumeMode == corev1.PersistentVolumeBlock {
 			accessMode = corev1.ReadWriteMany
 		}
+
 		By("Creating DataVolume")
-		dvBlock := libstorage.NewBlankDataVolume(util.NamespaceTestDefault, sc, "64Mi", accessMode, volumeMode)
-		_, err := virtClient.CdiClient().CdiV1beta1().DataVolumes(dvBlock.Namespace).Create(context.Background(), dvBlock, metav1.CreateOptions{})
+		dvBlock := libdv.NewDataVolume(
+			libdv.WithBlankImageSource(),
+			libdv.WithPVC(sc, "64Mi", accessMode, volumeMode),
```

Comment:
What about a new constant for blank images (`64Mi`)?

Reply:
:+1000